calicoctl/commands/resources.go

```diff
@@ -188,6 +189,21 @@ func executeConfigCommand(args map[string]interface{}, action action) commandRes
 			results.err = err
 			break
 		}
+
+		for i, _ := range res {
+			// Remove the cluster specific metadata if the "--export" flag is specified
+			if argutils.ArgBoolOrFalse(args, "--export") {
+				rom := res[i].(v1.ObjectMetaAccessor).GetObjectMeta()
+				rom.SetNamespace("")
```

Comment:
didn't know k8s also removes the namespace with `--export`, but I just checked it and it does, should be ok because both of our namespaced resources are generated from k8s resources